# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1418304523 -3600
#      Thu Dec 11 14:28:43 2014 +0100
# Node ID 50d4fd16fb3a1f0ef7205303cfe344d81fcbd4d2
# Parent  fb09907f4e49023bfc7f1cdc0349128c04af9296
scripts/general/module.mk: new entry for nargchk.m and alphabetized. (bug #43798)

diff --git a/scripts/general/module.mk b/scripts/general/module.mk
--- a/scripts/general/module.mk
+++ b/scripts/general/module.mk
@@ -34,28 +34,29 @@ general_FCN_FILES = \
   general/flipud.m \
   general/gradient.m \
   general/idivide.m \
   general/inputParser.m \
   general/int2str.m \
   general/interp1.m \
   general/interp2.m \
   general/interp3.m \
+  general/interpft.m \
   general/interpn.m \
-  general/interpft.m \
   general/isdir.m \
   general/isequal.m \
   general/isequaln.m \
   general/loadobj.m \
   general/logspace.m \
   general/methods.m \
+  general/nargchk.m \
   general/narginchk.m \
   general/nargoutchk.m \
+  general/nextpow2.m \
   general/nthargout.m \
-  general/nextpow2.m \
   general/num2str.m \
   general/pol2cart.m \
   general/polyarea.m \
   general/postpad.m \
   general/prepad.m \
   general/profexplore.m \
   general/profile.m \
   general/profshow.m \
@@ -69,18 +70,18 @@ general_FCN_FILES = \
   general/rotdim.m \
   general/saveobj.m \
   general/shift.m \
   general/shiftdim.m \
   general/sortrows.m \
   general/sph2cart.m \
   general/structfun.m \
   general/subsindex.m \
+  general/trapz.m \
   general/triplequad.m \
-  general/trapz.m \
   general/validateattributes.m \
   $(general_PRIVATE_FCN_FILES)
 
 FCN_FILES += $(general_FCN_FILES)
 
 PKG_ADD_FILES += general/PKG_ADD
 
 DIRSTAMP_FILES += general/$(octave_dirstamp)
